Detectors/TPC/qc/src/PID.cxx

```diff
@@ -153,7 +171,20 @@ bool PID::processTrack(const o2::tpc::TrackTPC& track)
     if (std::abs(tgl) < mCutAbsTgl) {
       mMapHist["hdEdxTotVsp"][idEdxType]->Fill(pTPC, dEdxTot[idEdxType]);
       mMapHist["hdEdxMaxVsp"][idEdxType]->Fill(pTPC, dEdxMax[idEdxType]);
+      for (size_t iSpecies = 0; iSpecies < o2::track::PID::NIDs; ++iSpecies) {
+        dEdxRatio = dEdxTot[idEdxType] / mPID.getExpectedSignal(track, static_cast<o2::track::PID::ID>(iSpecies));
+      }
+      const auto PIDHypothesis = mPID.getMostProbablePID(track, mCutKrangeMin, mCutKrangeMax, mCutPrangeMin, mCutPrangeMax, mCutDrangeMin, mCutDrangeMax, mCutTrangeMin, mCutTrangeMax, 1, mPIDSigma);
+      int Hypothesis = static_cast<int>(PIDHypothesis) + 1;
+      if (static_cast<int>(PIDHypothesis) <= o2::track::PID::NIDs) {
```

Comment:
Not sure whey the pid hypothesis is calculated here again. The goal should be to use the one carried out during the reconstruction. So I guess this should b (+ naming conventions):
```suggestion
      const auto pidHypothesis = track.getPID().getID();
      if (pidHypothesis <= o2::track::PID::NIDs) {
```
and remove the unnecessary variables added above.